Trigger/Algo/src/Trigger/TrigMipReco.cxx

```diff
@@ -0,0 +1,300 @@
+#include "Trigger/TrigMipReco.h"
+
+namespace trigger {
+
+void TrigMipReco::onNewRun(const ldmx::RunHeader& rh) {
+  profiling_map_["processing_time_"] = 0;
+}
+
+void TrigMipReco::configure(framework::config::Parameters& ps) {
+  hit_coll_name_ = ps.getParameter<std::string>("hit_coll_name");
+  pass_coll_name_ = ps.getParameter<std::string>("pass_coll_name");
+  hit_coll_passname_ = ps.getParameter<std::string>("hit_coll_passname");
+  calorimeter_type_is_hcal_ = ps.getParameter<bool>("calorimeter_type_is_hcal");
+
+  max_layer_ = ps.getParameter<int>("max_layer", 32);
+  search_radius_ = ps.getParameter<float>("search_radius", 50.0f);  // mm
+  min_track_length_ = ps.getParameter<int>("min_track_length", 5);
+  isolation_e_cut_ = ps.getParameter<float>("isolation_e_cut",
+                                            180.0f);  // MeV; Change as needed
```

Comment:
Our format is forcing the lines to be 80 characters long. On lines like these, it is helpful to put the comment above the line so that the code can look prettier.

More of an aesthetic choice though so you can ignore this if you don't have the time.